src/ansys/tools/repo_sync/__main__.py

```diff
@@ -1,7 +1,7 @@
 """Tool to copy the content of one repo toward an other.
 Run with:
 
-repo-sync -o Organization -r repository -b branch_name -p path_to_protos_directory
+repo-sync -o Organization -r repository --from-dir ... --to-dir ... --token ... [-b branch_name -m manifest_file -d]
```

Comment:
I cannot remember why I used `Organization` instead of `organization`.

```suggestion
repo-sync -o organization -r repository --from-dir ... --to-dir ... --token ... [-b branch_name -m manifest_file -d]
```